src/components/v2/V2Project/V2ProjectReconfigureModal/index.tsx

```diff
@@ -160,13 +163,24 @@ export default function V2ProjectReconfigureModal({
                 Changes to project details will take effect immediately.
               </Trans>
             </p>
-            <ReconfigureButton
```

Comment:
only layout changes to this component—adding new section for configuring project handle